gh-pages/Gemfile.lock

```diff
@@ -30,10 +30,12 @@ GEM
     ethon (0.12.0)
       ffi (>= 1.3.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```

Comment:
I guess these files were required to get it working locally while not being required otherwise? I would therefore remove them, if that's correct.

Reply:
Yes,you're right! I removed it :)